src/core/client/stream/constants.ts

```diff
@@ -4,7 +4,7 @@ export const VIEWER_STATUS_CONTAINER_ID = "viewer-status";
 
 export const POST_COMMENT_FORM_ID = "post-comment-form";
 
-export const MAX_REPLY_INDENT_DEPTH = 7;
+export const MAX_REPLY_INDENT_DEPTH = 8;
```

Comment:
I don't think making this 8 on its own will work. You have to make it 8 and subtract one elsewhere or leave this as 7 and subtract one in the cases where it needs to be one less to check when it should and shouldn't show things.